.travis.yml

```diff
@@ -80,50 +80,72 @@ matrix:
     # The latest stable and head versions of ruby built by clang on OSX
     - os: osx
       compiler: clang
-      rvm: 2.2
+      rvm: ruby-head-clang
```

Comment:
What was the reason behind these changes? Can you please try to write comments for your changes and what issues it is solving?

Reply:
For osx builds, the clang version of corresponding ruby versions are used. Additional parameters are added so as to make sure that no error occurs during build. I'll add the comments accordingly.

Reply:
Yeah!  That I understood. But I believe without also it is fine. Anyways it is not a problem.